runtime/doc/news.txt

```diff
@@ -86,6 +86,13 @@ EVENTS
 
 • |vim.ui_attach()| callbacks for |ui-messages| `msg_show` events are executed in
   |api-fast| context.
+• Various additions for the following UI events:
```

Comment:
```suggestion
• New fields in these existing UI events:
```

Reply:
What about:
"New/enhanced arguments in these existing UI events:`